src/common/components/paymentMethods/creditCardForm/creditCardForm.component.js

```diff
@@ -138,7 +138,7 @@ class CreditCardController {
           }) // Send masked card number when card number is not updated
         : this.tsysService.getManifest()
           .mergeMap(data => {
-            const productionEnvironments = ['production', 'prodcloud']
+            const productionEnvironments = ['production', 'prodcloud', 'preprod']
```

Comment:
Could you add a test to ensure it handles the `preprod` environment properly?

Also might be good to add a test here as well: src/common/components/paymentMethods/bankAccountForm/bankAccountForm.component.spec.js